homeassistant/components/matter/valve.py

```diff
@@ -52,9 +52,14 @@ async def async_close_valve(self) -> None:
 
     async def async_set_valve_position(self, position: int) -> None:
         """Move the valve to a specific position."""
-        await self.send_device_command(
-            ValveConfigurationAndControl.Commands.Open(targetLevel=position)
-        )
+        if position > 0:
+            await self.send_device_command(
+                ValveConfigurationAndControl.Commands.Open(targetLevel=position)
+            )
+        else:
+            await self.send_device_command(
+                ValveConfigurationAndControl.Commands.Close()
+            )
```

Comment:
Nit-pick: HA usually prefers code be less indented when possible

```suggestion
            return
        await self.send_device_command(
            ValveConfigurationAndControl.Commands.Close()
        )
```